### ISSUE TITLE
Some Windows are not visible on the preview screen

### ISSUE THREAD

**Issue by Slider-Override** [newlooper/VirtualSpace#132]
If i have for example the Taskmanager open I did not see it on the preview to move it over to another Virtual Desktop.

Running Win11 24H2 with Build-0.2.1104

I did not get the full details, yet, but its makes no differenz if i run VirtualSpace in Administrator mode or not,
If I run a CMD as Administrator i see it but some windows not,

I did not see:
 - Taskmanager
 - msconfig
 - RemoteDesktopManager
 - PowerToy

If someone has the same issue or a idea to fix please let me know

**Comment by Slider-Override** [comment]
smal update on this:

The issue is still present.

You can reproduse the issue by opening "Windows PowerShell ISE" its enoth, its for any reason not visible on the GRID overview (Win+TAB) :-(

**Comment by Slider-Override** [comment]
Hmmmm Issue is still present but I am not sure why, maybe its just on 24H2, I am the only one with the issue ?

**Comment by Slider-Override** [comment]
still a issue, something is wrong here

**Comment by Slider-Override** [comment]
The issues are still present on the latest Build: "VirtualSpace_0.2.1412.974"

Any idea how to fix this ?
I see more and more Applications which are not visible on the overview screen.

- Powershell GUIs also missing

**Comment by newlooper** [comment]
It might be related to the program's execution permissions，was VS run as admin?
The windows you mentioned are all visible in my VS MainView (VS run as admin).

**Comment by Slider-Override** [comment]
Yes it runs as admin, but the msconfig for example is not visible
same for my Powershell GUIs:

Try this Powershell if you see the GUI in the overview:

`Add-Type -AssemblyName System.Windows.Forms; $f = New-Object Windows.Forms.Form; $f.Text = 'Hallo Newlooper'; $l = New-Object Windows.Forms.Label; $l.Text = 'simple GUI'; $l.AutoSize = $true; $f.Controls.Add($l); $f.ShowDialog()`
